resources/sdk/purecloudjavascript/templates/README.mustache

```diff
@@ -189,6 +189,81 @@ client.setAccessToken(yourAccessToken);
 
 When authenticating in a browser using `loginClientCredentialsGrant(...)`, if the user completes the authentication process but their session is unable to be authorized, they will still be redirected back to the redirect URI. When `loginClientCredentialsGrant(...)` is invoked after the failure redirect, the promise returned will be rejected with an error message built from the `error` and `error_description`. The error information, as well as the state, can be accessed via `platformClient.ApiClient.instance.authData`. The application is expected to identify these login failures and interact with the user in a manner appropriate for the application.
 
+## SDK Logging
+
+Logging of API requests and responses can be controlled by a number of parameters on the `Configuration`'s `Logger` instance.
+
+`log_level` values:
+1. LTrace (HTTP Method, URL, Request Body, HTTP Status Code, Request Headers, Response Headers)
+2. LDebug (HTTP Method, URL, Request Body, HTTP Status Code, Request Headers)
+3. LError (HTTP Method, URL, Request Body, Response Body, HTTP Status Code, Request Headers, Response Headers)
+4. LNone - default
+
+`log_format` values:
+1. JSON
+2. TEXT - default
+
+By default, the request and response bodies are not logged because these can contain PII. Be mindful of this data if choosing to log it.  
+To log to a file, provide a `log_file_path` value. SDK users are responsible for the rotation of the log file.
+
+Example logging configuration:
+```{"language":"javascript"}
+client.config.logger.log_level = client.config.logger.logLevelEnum.level.LTrace;
```

Comment:
Tried running this and I get the following error, and a similar one for the log_format:
client.config.logger.log_level = client.config.logger.logLevelEnum.level.LTrace;
                                                                   ^
TypeError: Cannot read property 'level' of undefined

Reply:
Added a getter for those enums. Should be working now.